src/main/java/com/uwetrottmann/tmdb2/Tmdb.java

```diff
@@ -17,14 +51,16 @@
 import retrofit2.Retrofit;
 import retrofit2.converter.gson.GsonConverterFactory;
 
+import java.io.IOException;
+
 /**
  * Helper class for easy usage of the TMDB v3 API using retrofit.
- *
+ * <p>
```

Comment:
It was done by the auto Code Formatting of my Intellij Idea, i will take care of it, Whatsoever, on an Empty line, in my java docs, line is not getting changed, with <p> it does.